docs/performance/inserts/sequences.rst

```diff
@@ -0,0 +1,196 @@
+.. _autogenerated_sequences_performance:
+
+###########################################################
+ Autogenerated sequences and PRIMARY KEY values in CrateDB
+###########################################################
+
+As you begin working with CrateDB, you might be puzzled why CrateDB does not
+have a built-in, auto-incrementing "serial" data type as PostgreSQL or MySQL.
+
+As a distributed database, designed to scale horizontally, CrateDB needs as
+many operations as possible to complete independently on each node without
+any coordination between nodes.
+
+Maintaining a global auto-increment value requires that a node checks with 
+other nodes before allocating a new value. This bottleneck would be hindering
+our ability to achieve `extremely fast ingestion speeds`_.
+
+That said, there are many alternatives available and we can also implement true
+consistent/synchronized sequences if we want to.
+
+************************************
+ Using a timestamp as a primary key
+************************************
+
+This option involves declaring a column as follows:
+
+.. code:: psql
+
+   BIGINT DEFAULT now() PRIMARY KEY
+
+:Pros: Always increasing number - ideal if we need to timestamp records
+creation anyway
+
+
+:Cons: gaps between the numbers, not suitable if we may have more than one
+record on the same millisecond
```

Comment:
This snippet causes the _WARNING: Field list ends without a blank line; unexpected unindent._

-- https://github.com/crate/cratedb-guide/actions/runs/14473917871/job/40594700368?pr=185#step:4:581

I think this might work better:
```suggestion
:Pros: Always increasing number - ideal if we need to timestamp records
       creation anyway
:Cons: gaps between the numbers, not suitable if we may have more than one
       record on the same millisecond
```